package.json

```diff
@@ -5,10 +5,11 @@
   "main": "index.js",
   "scripts": {
     "build": "lerna exec --parallel -- babel src -d dist --ignore '**/*.test.jsx,**/*.stories.jsx' --root-mode upward",
+    "ci:packages": "lerna exec -- npm ci",
     "clear_styled_components": "for d in ./packages/*/*; do rm -rf $d/node_modules/styled-components; done; echo \"Cleared nested packages/**/node_modules/styled-components\"",
     "deploy-storybook": "build-storybook -o storybook_dist && gh-pages -d storybook_dist",
     "install:packages": "lerna exec -- npm install",
-    "postinstall": "npm run install:packages",
+    "postinstall": "npm run ci:packages",
```

Comment:
IMO we should remain doing `npm run ci:packages` as a post install step here, Until we decide and create a process for ensuring packages don't go stale

Reply:
We could leave this as it is and do `make install` => `npm ci --ignore-scripts; npm run ci:packages`

Reply:
`--ignore-scripts;` is unusable -- I believe it ignores the postinstall scripts of dependencies, too.

Reply:
Just to summarise the arguments put forward elsewhere:

* We could explore changing the postinstall step to something like `if !ENV.IS_CI; do npm run install:packages; done`, but we like knowing that our local env is the same as CI env, & that tests will produce the same results.
* The "obviously it doesn't scale" refers to: Keeping `install:packages` as the postinstall step locally will result in version bumps in packages that you aren't currently working on. When someone wishes to make changes to `psammead-copyright`, & `psammead-headlines` has received new dependency versions then they have to be careful not to commit those changes, & those changes may cause them issues if they have accidentally introduced errors or behaviour changes.

Reply:
So the summary is: we're sticking with `npm ci`?